config.json

```
@@ -420,6 +420,17 @@
           "overflow",
           "integral-numbers"
         ]
+      },
+      {
+        "slug": "log-analysis",
+        "name": "Log Analysis",
+        "uuid": "4a940e54-eda8-4500-bcb2-25d83af01fb1",
+        "concepts": ["extension-methods"],
+        "prerequisites": [
+          "collections",
+          "strings",
+          "booleans"
```

I think we just need `strings` as a prerequisite with the current setup.

```suggestion
          "strings"
```